### ISSUE TITLE
Warnings in 2021.12b0

### ISSUE THREAD

**Issue by tomlut** [briis/smartweather#87]
Just updated to the 2021.12b0 Home Assistant beta version and am getting the following warnings:
```
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:543
First occurred: 11:50:01 (2960 occurrences)
Last logged: 11:51:26

Entity sensor.smartweather_lightning_last_1hrs (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.smartweather_lightning_last_3hrs (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.smartweather_battery_st_00041965 (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity weather.smartweather_timbertop_hourly (<class 'custom_components.smartweather.weather.SmartWeatherWeather'>) implements device_state_attributes. Please report it to the custom component author.
Entity weather.smartweather_timbertop_daily (<class 'custom_components.smartweather.weather.SmartWeatherWeather'>) implements device_state_attributes. Please report it to the custom component author.
```

https://rc.home-assistant.io/blog/2021/12/03/release-202112/#updates-for-custom-integration-developers

**Comment by tomlut** [comment]
```
021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_temperature (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_feels_like (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_dewpoint (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_wind_speed (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_wind_direction (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_wind_gust (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_rain_today (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_rain_rate (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_rain_last_hour (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_humidity (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_sea_level_pressure (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_uv (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_solar_radiation (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_brightness (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_lightning_count (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_lightning_distance (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_lightning_time (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_lightning_last_1hrs (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_lightning_last_3hrs (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component author.
2021-12-04 12:20:55 WARNING (MainThread) [homeassistant.helpers.entity] Entity sensor.smartweather_battery_st_00041965 (<class 'custom_components.smartweather.sensor.SmartWeatherSensor'>) implements device_state_attributes. Please report it to the custom component 
```

**Comment by briis** [comment]
Thanks for reporting this. This integration will be deprecated, as I have written a completely new one, that is easier to maintain and adheres to Home Assistant code practices.
I will fix this issue, but if you want to already now use the new Integration you can go here: [WeatherFlow Weather](https://github.com/briis/hass-weatherflow). I have submitted a PR to get this in the Default HACS store, but that is still waiting to be merged, son until then you will have to add as a Custom Repository (See the README file)

**Comment by briis** [comment]
I have released 2.2.12, which fixes the above.

**Comment by tomlut** [comment]
Thanks for the quick fix. I'll check out the new integration.
